launch/elasticsearch-toolbox.yml

```diff
@@ -18,3 +18,23 @@ dependencies:
 shepherds:
 - ryan.burns@clever.com
 team: eng-infra
+alarms:
+- type: InternalErrorAlarm
+  severity: minor
+  # channel: this will go to the channel that this team has configured in catapult for minor alarms
+  parameters:
+    threshold: 0.01
+  extraParameters:
+    source: Target
+- type: InternalErrorAlarm
+  severity: major
+  # channel: this will go to the channel that this team has configured in catapult for major alarms
+  parameters:
+    threshold: 0.05
+  extraParameters:
+    source: Target
+- type: InternalErrorAlarm
+  severity: major
+  # channel: this will go to the channel that this team has configured in catapult for major alarms
+  parameters:
+    threshold: 0.01
```

Comment:
This feels weird, why would 1% of target requests be minor, but 1% of all requests be major?